index.html

```diff
@@ -1,18 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <!-- displays site properly based on user's device -->
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
   <link rel="stylesheet" href="style.css">
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <!-- displays site properly based on user's device -->
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
  <link rel="stylesheet" href="style.css">
  <title>Advice generator</title>
</head>
<body>

  <!-- Main -->
  <main>

    <!-- Advice Container -->
    <div class="advice-container">

      <!-- Advice ID goes here -->
      <p class="advice-id">
        Advice #<span id="advice-number"></span>
      </p>

      <!-- Advice text goes here -->
      <p class="advice-text">
        Loading..
      </p>
    </div>

    <!-- Divider -->
    <img src="images/pattern-divider-desktop.svg" alt="" class="divider">
    
    <!-- Dice Button -->
    <button id="dice"><img src="images/icon-dice.svg" alt="Dice to generate new advice"></button>

  </main>

  <!-- Attribution -->
  <footer class="attribution">
    Coded by 
    <a href="https://instagram.com/developer_rahul_">
     Developer Rahul
    </a>.
  </footer>

  <!-- Script -->
  <script src="script.js"></script>

</body>
</html>